### ISSUE TITLE
Rendering errors

### ISSUE THREAD

**Issue by Romz24** [jbredwards/Fluidlogged-API#13]
After I installed your mod, I am constantly flooded into the chat. Can you fix this somehow?

![image](https://user-images.githubusercontent.com/2458113/123467169-de1d4d00-d5f8-11eb-8ef0-76df31ce9260.png)

```
[20:44:31] [Chunk Batcher 13/ERROR] [CodeChickenLib]: 
 CCL has caught an exception whilst rendering a block
  BlockPos:      x:2154, y:28, z:3824
  Block Class:   class net.minecraft.block.BlockStaticLiquid
  Registry Name: minecraft:lava
  Metadata:      10
  State:         minecraft:lava[level=10]
 Tile at position
  Tile Class:    null
  Tile Id:       null
  Tile NBT:      null
You can turn off player messages in the CCL config file.

net.minecraft.util.ReportedException: Tesselating block in world
	at net.minecraft.client.renderer.BlockRendererDispatcher.renderBlock(BlockRendererDispatcher.java:95) ~[bvm.class:?]
	at codechicken.lib.render.block.CCBlockRendererDispatcher.renderBlock(CCBlockRendererDispatcher.java:82) [CCBlockRendererDispatcher.class:?]
	at git.jbredwards.fluidlogged_api.client.util.FluidloggedBlockRendererDispatcher.renderBlock(FluidloggedBlockRendererDispatcher.java:36) [FluidloggedBlockRendererDispatcher.class:?]
	at mrtjp.projectred.relocation.MovingBlockRenderDispatcher.renderBlock(renders.scala:277) [MovingBlockRenderDispatcher.class:?]
	at net.minecraft.client.renderer.chunk.RenderChunk.rebuildChunk(RenderChunk.java:328) [bxr.class:?]
	at net.minecraft.client.renderer.chunk.ChunkRenderWorker.processTask(SourceFile:100) [bxn.class:?]
	at net.minecraft.client.renderer.chunk.ChunkRenderWorker.run(SourceFile:43) [bxn.class:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_281]
Caused by: java.lang.IllegalArgumentException: Cannot set unlisted property net.minecraftforge.common.property.PropertyFloat@67553940 to NaN on block minecraft:lava, it is not an allowed value
	at net.minecraftforge.common.property.ExtendedBlockState$ExtendedStateImplementation.withProperty(ExtendedBlockState.java:117) ~[ExtendedBlockState$ExtendedStateImplementation.class:?]
	at git.jbredwards.fluidlogged_api.asm.swapper.BlockLiquidBase.getExtendedState(BlockLiquidBase.java:107) ~[BlockLiquidBase.class:?]
	at net.minecraft.client.renderer.BlockRendererDispatcher.renderBlock(BlockRendererDispatcher.java:79) ~[bvm.class:?]
	... 7 more
[20:44:31] [Chunk Batcher 13/INFO] [net.minecraft.client.gui.GuiNewChat]: [CHAT] CCL Caught an exception rendering a block. See the log for info.
```

**Comment by jbredwards** [comment]
fixed in v1.4.3!

**Comment by Romz24** [comment]
I am using v1.4.4 version, the error still occurs.

```
[19:41:34] [Chunk Batcher 11/ERROR] [CodeChickenLib]: 
 CCL has caught an exception whilst rendering a block
  BlockPos:      x:-331, y:38, z:-720
  Block Class:   class net.minecraft.block.BlockStaticLiquid
  Registry Name: minecraft:water
  Metadata:      12
  State:         minecraft:water[level=12]
 Tile at position
  Tile Class:    null
  Tile Id:       null
  Tile NBT:      null
You can turn off player messages in the CCL config file.

net.minecraft.util.ReportedException: Tesselating block in world
	at net.minecraft.client.renderer.BlockRendererDispatcher.renderBlock(BlockRendererDispatcher.java:95) ~[bvm.class:?]
	at codechicken.lib.render.block.CCBlockRendererDispatcher.renderBlock(CCBlockRendererDispatcher.java:82) [CCBlockRendererDispatcher.class:?]
	at git.jbredwards.fluidlogged_api.client.util.FluidloggedBlockRendererDispatcher.renderBlock(FluidloggedBlockRendererDispatcher.java:38) [FluidloggedBlockRendererDispatcher.class:?]
	at mrtjp.projectred.relocation.MovingBlockRenderDispatcher.renderBlock(renders.scala:277) [MovingBlockRenderDispatcher.class:?]
	at net.minecraft.client.renderer.chunk.RenderChunk.rebuildChunk(RenderChunk.java:328) [bxr.class:?]
	at net.minecraft.client.renderer.chunk.ChunkRenderWorker.processTask(SourceFile:100) [bxn.class:?]
	at net.minecraft.client.renderer.chunk.ChunkRenderWorker.run(SourceFile:43) [bxn.class:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_281]
Caused by: java.lang.IllegalArgumentException: Cannot set unlisted property net.minecraftforge.common.property.PropertyFloat@44a98a23 to NaN on block minecraft:water, it is not an allowed value
	at net.minecraftforge.common.property.ExtendedBlockState$ExtendedStateImplementation.withProperty(ExtendedBlockState.java:117) ~[ExtendedBlockState$ExtendedStateImplementation.class:?]
	at git.jbredwards.fluidlogged_api.asm.swapper.BlockLiquidBase.getExtendedState(BlockLiquidBase.java:115) ~[BlockLiquidBase.class:?]
	at net.minecraft.client.renderer.BlockRendererDispatcher.renderBlock(BlockRendererDispatcher.java:79) ~[bvm.class:?]
	... 7 more
[19:41:34] [Chunk Batcher 11/ERROR] [CodeChickenLib]: 
 CCL has caught an exception whilst rendering a block
  BlockPos:      x:-332, y:38, z:-719
  Block Class:   class net.minecraft.block.BlockStaticLiquid
  Registry Name: minecraft:water
  Metadata:      12
  State:         minecraft:water[level=12]
 Tile at position
  Tile Class:    null
  Tile Id:       null
  Tile NBT:      null
You can turn off player messages in the CCL config file.

net.minecraft.util.ReportedException: Tesselating block in world
	at net.minecraft.client.renderer.BlockRendererDispatcher.renderBlock(BlockRendererDispatcher.java:95) ~[bvm.class:?]
	at codechicken.lib.render.block.CCBlockRendererDispatcher.renderBlock(CCBlockRendererDispatcher.java:82) [CCBlockRendererDispatcher.class:?]
	at git.jbredwards.fluidlogged_api.client.util.FluidloggedBlockRendererDispatcher.renderBlock(FluidloggedBlockRendererDispatcher.java:38) [FluidloggedBlockRendererDispatcher.class:?]
	at mrtjp.projectred.relocation.MovingBlockRenderDispatcher.renderBlock(renders.scala:277) [MovingBlockRenderDispatcher.class:?]
	at net.minecraft.client.renderer.chunk.RenderChunk.rebuildChunk(RenderChunk.java:328) [bxr.class:?]
	at net.minecraft.client.renderer.chunk.ChunkRenderWorker.processTask(SourceFile:100) [bxn.class:?]
	at net.minecraft.client.renderer.chunk.ChunkRenderWorker.run(SourceFile:43) [bxn.class:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_281]
Caused by: java.lang.IllegalArgumentException: Cannot set unlisted property net.minecraftforge.common.property.PropertyFloat@44a98a23 to NaN on block minecraft:water, it is not an allowed value
	at net.minecraftforge.common.property.ExtendedBlockState$ExtendedStateImplementation.withProperty(ExtendedBlockState.java:117) ~[ExtendedBlockState$ExtendedStateImplementation.class:?]
	at git.jbredwards.fluidlogged_api.asm.swapper.BlockLiquidBase.getExtendedState(BlockLiquidBase.java:115) ~[BlockLiquidBase.class:?]
	at net.minecraft.client.renderer.BlockRendererDispatcher.renderBlock(BlockRendererDispatcher.java:79) ~[bvm.class:?]
	... 7 more
```

**Comment by jbredwards** [comment]
I thought I fixed this in v1.5, try that version and let me know if you're still experiencing the issue
